pages/common/uuencode.md

```diff
@@ -0,0 +1,16 @@
+# uuencode
+
+> Encode binary files into ASCII for transport via mediums that only support simple ASCII encoding.
+> More information: <https://manned.org/uuencode>.
+
+- Encode a file and print the result to stdout:
+
+`uuencode {{path/to/input_file}} {{output_file_name_after_decoding}}`
+
+- Encode a file and and write the result to a file:
+
+`uuencode -o {{path/to/output_file}} {{path/to/input_file}} {{output_file_name_after_decoding}}`
+
+- Encode a file in Base64 instead of the tradition uuencode encoding and and write the result to a file:
```

Comment:
```suggestion
- Encode a file in Base64 instead of the traditional uuencode encoding and and write the result to a file:
```
With `tratitional uuencode oncoding` you mean ASCII, right?

Reply:
Both formats are technically ASCII. Here's an example:
```console
$ uuencode pages.zh/common/gpg.md gpgmd
begin 644 gpgmd
M(R!G<&<*"CX@1TY5(%!R:79A8WD@1W5A<F3C@((*/B#HK[?ECX+IF(5@9W!G
M,F#DNH;HIZ,@1TY5(%!R:79A8WD@1W5A<F0@,N.`@@H^(.:;M.6DFN2_H>:!
MK^^\FCQH='1P<SHO+V1O8W,N<F5L96YG+FQI;G5X9F]U;F1A=&EO;BYO<F<O
M96XO;&%T97-T+V=P9RYH=&UL/N.`@@H*+2#DN(WEBJ#EKX;OO(SDNX7EK[E@
M9&]C+G1X=&#HOYOHH8SGK;[ED(WOO(CGE)_FB)!@9&]C+G1X="YA<V-@[[R,
MYJ"\Y;R/Y+BZ($%30TE)(.>@@>6]HN6\C^^\B>^\F@H*8&=P9R`M+6-L96%R
M<VEG;B![>V1O8RYT>'1]?6`*"BT@Y:^Y8&1O8RYT>'1@Z+^;Z*&,YZV^Y9"-
MY;FVY8J@Y:^&[[R(YY2?YHB08&1O8RYT>'0N87-C8.^\C.:@O.6\C^2XNB!!
M4T-)22#GH('EO:+EO(_OO(GOO)H*"F!G<&<@+2UL;V-A;"UU<V5R('M[<V5N
M9&5R7VED?7T@+2UR96-I<&EE;G0@>WMR96-I<&EE;G1?:61]?2`M+6%R;6]R
M("TM<VEG;B`M+65N8W)Y<'0@>WMD;V,N='AT?7U@"@HM(.2XNN:.I>:4MNB`
MA2!A;&EC94!E>&%M<&QE+F-O;2#EBJ#EKX9@9&]C+G1X=&#OO(CGE)_FB)!@
M9&]C+G1X="YG<&=@[[R)[[R:"@I@9W!G("TM96YC<GEP="`M+7)E8VEP:65N
M="![>V%L:6-E0&5X86UP;&4N8V]M?7T@>WMD;V,N='AT?7U@"@HM(.6/JN>4
MJ.6OAN>@@>6*H.6OAF!D;V,N='AT8.^\B.>4G^:(D&!D;V,N='AT+F=P9V#O
MO(GOO)H*"F!G<&<@+2US>6UM971R:6,@>WMD;V,N='AT?7U@"@HM(.BGH^6O
MAF!D;V,N='AT+F=P9V#OO(CHOI/EA[KEB+#FH(?EAX;HOI/EA[KOO(GOO)H*
M"F!G<&<@+2UD96-R>7!T('M[9&]C+G1X="YG<&=]?6`*"BT@Y:^\Y86EY+B`
MY+BJY86LZ9*E[[R:"@I@9W!G("TM:6UP;W)T('M[<'5B;&EC+F=P9WU]8`H*
M+2#EK[SEA[H@86QI8V5`97AA;7!L92YC;VT@YYJ$Y86LZ9*E[[R(Z+Z3Y8>Z
MY8BPYJ"'Y8>&Z+Z3Y8>Z[[R)[[R:"@I@9W!G("TM97AP;W)T("TM87)M;W(@
M>WMA;&EC94!E>&%M<&QE+F-O;7U]8`H*+2#EK[SEA[H@86QI8V5`97AA;7!L
M92YC;VT@YYJ$YZ>!Z9*E[[R(Z+Z3Y8>ZY8BPYJ"'Y8>&Z+Z3Y8>Z[[R)[[R:
M"@I@9W!G("TM97AP;W)T+7-E8W)E="UK97ES("TM87)M;W(@>WMA;&EC94!E
.>&%M<&QE+F-O;7U]8`H`
`
end

$ uuencode -m pages.zh/common/gpg.md gpgmd
begin-base64 644 gpgmd
IyBncGcKCj4gR05VIFByaXZhY3kgR3VhcmTjgIIKPiDor7flj4LpmIVgZ3BnMmDkuobop6MgR05V
IFByaXZhY3kgR3VhcmQgMuOAggo+IOabtOWkmuS/oeaBr++8mjxodHRwczovL2RvY3MucmVsZW5n
LmxpbnV4Zm91bmRhdGlvbi5vcmcvZW4vbGF0ZXN0L2dwZy5odG1sPuOAggoKLSDkuI3liqDlr4bv
vIzku4Xlr7lgZG9jLnR4dGDov5vooYznrb7lkI3vvIjnlJ/miJBgZG9jLnR4dC5hc2Ng77yM5qC8
5byP5Li6IEFTQ0lJIOeggeW9ouW8j++8ie+8mgoKYGdwZyAtLWNsZWFyc2lnbiB7e2RvYy50eHR9
fWAKCi0g5a+5YGRvYy50eHRg6L+b6KGM562+5ZCN5bm25Yqg5a+G77yI55Sf5oiQYGRvYy50eHQu
YXNjYO+8jOagvOW8j+S4uiBBU0NJSSDnoIHlvaLlvI/vvInvvJoKCmBncGcgLS1sb2NhbC11c2Vy
IHt7c2VuZGVyX2lkfX0gLS1yZWNpcGllbnQge3tyZWNpcGllbnRfaWR9fSAtLWFybW9yIC0tc2ln
biAtLWVuY3J5cHQge3tkb2MudHh0fX1gCgotIOS4uuaOpeaUtuiAhSBhbGljZUBleGFtcGxlLmNv
bSDliqDlr4ZgZG9jLnR4dGDvvIjnlJ/miJBgZG9jLnR4dC5ncGdg77yJ77yaCgpgZ3BnIC0tZW5j
cnlwdCAtLXJlY2lwaWVudCB7e2FsaWNlQGV4YW1wbGUuY29tfX0ge3tkb2MudHh0fX1gCgotIOWP
queUqOWvhueggeWKoOWvhmBkb2MudHh0YO+8iOeUn+aIkGBkb2MudHh0LmdwZ2DvvInvvJoKCmBn
cGcgLS1zeW1tZXRyaWMge3tkb2MudHh0fX1gCgotIOino+WvhmBkb2MudHh0LmdwZ2DvvIjovpPl
h7rliLDmoIflh4bovpPlh7rvvInvvJoKCmBncGcgLS1kZWNyeXB0IHt7ZG9jLnR4dC5ncGd9fWAK
Ci0g5a+85YWl5LiA5Liq5YWs6ZKl77yaCgpgZ3BnIC0taW1wb3J0IHt7cHVibGljLmdwZ319YAoK
LSDlr7zlh7ogYWxpY2VAZXhhbXBsZS5jb20g55qE5YWs6ZKl77yI6L6T5Ye65Yiw5qCH5YeG6L6T
5Ye677yJ77yaCgpgZ3BnIC0tZXhwb3J0IC0tYXJtb3Ige3thbGljZUBleGFtcGxlLmNvbX19YAoK
LSDlr7zlh7ogYWxpY2VAZXhhbXBsZS5jb20g55qE56eB6ZKl77yI6L6T5Ye65Yiw5qCH5YeG6L6T
5Ye677yJ77yaCgpgZ3BnIC0tZXhwb3J0LXNlY3JldC1rZXlzIC0tYXJtb3Ige3thbGljZUBleGFt
cGxlLmNvbX19YAo=
====

$
```

Reply:
What actually is `tratitional uuencode encoding`? Is there maybe a better word for it?

Reply:
Well, we could call it `ugly encoding`.

Reply:
Just kidding.  It's definitely not pretty though 😄 

Maybe "default uuencode encoding"? I don't really know if it's a standard thing.

Reply:
I think `default` makes more sense than `tratitional`.

Reply:
I do too.  I copied it verbatim from the man page, so blame them 😆
```suggestion
- Encode a file using Base64 instead of the default uuencode encoding and and write the result to a file:
```

Reply:
The wording as tratitional as the page itself 😆